src/imgui_ex/vcFileDialog.h

```diff
@@ -18,6 +18,7 @@ static const char *SupportedFileTypes_ProjectsImport[] = { ".json", ".udp" };
 
 static const char *SupportedTileTypes_ConvertExport[] = { ".uds" };
 static const char *SupportedTileTypes_QueryExport[] = { ".uds", ".las" };
+static const char *SupportedTileTypes_ExportDefault = ".las";
```

Comment:
I would recommend just using `SupportedTileTypes_QueryExport[0]` instead- the default should probably be `.uds` anyway

Reply:
Fixed.